java/src/main/java/org/rocksdb/FlinkEnv.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
Use the unified license defined in README ?

Reply:
resolved.